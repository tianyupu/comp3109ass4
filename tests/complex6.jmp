  if 1 goto L2;
  x = 5;
  return x;
L1:
  b = 2 - 1;
  if b goto L3;
  c = 2;
  return c;
L2:
  if a goto L1;
  a = a + c;
  return a;
L3:
  d = d + 1;
  d = 2;
  return d;
L4:
  a = 5;
  b = a * 3;
  return b;
