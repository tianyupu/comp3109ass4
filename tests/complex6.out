  if a goto L1;
  a = a + c;
  return a;
L1:
  return 2;
